tests/rustdoc-ui/issues/issue-120444-2.rs

```diff
@@ -0,0 +1,17 @@
+// compile-flags: --document-private-items
+
+#![deny(rustdoc::redundant_explicit_links)]
+
+pub mod webdavfs {
```

Comment:
Difference between ﻿`issue-120444-1` and ﻿`issue-120444-2`:
- In `﻿issue-120444-1`, it is declared as `﻿mod webavfs`;
- In ﻿`issue-120444-2`, it is declared as `﻿pub mod webavfs`